docs/setup-cluster/deploy-cluster/slurm/singularity.rst

```diff
@@ -182,3 +184,40 @@ You can view the current set of Docker image names in the cache with the ``-l``
       manage-singularity-cache -l
       determinedai/environments:py-3.8-pytorch-1.10-tf-2.8-cpu-096d730
       determinedai/environments:cuda-11.3-pytorch-1.10-tf-2.8-gpu-096d730
+
+.. _manage-enroot-cache:
+
+**********************************************************************
+ Managing the Enroot Image Cache using the manage-enroot-cache script
+**********************************************************************
+
+This script, ``/usr/bin/manage-enroot-cache``, simplifies the management of a set of shared Enroot
+.sqsh file downloads and then creates an Enroot container for use by the current user. It provides
+the following features:
+
+   -  Download the Determined default cuda, cpu, or rocm environment images
+   -  Download an arbitrary Docker image reference
+   -  Share a directory of re-usable imported .sqsh files
+   -  Create a per-user container from a shared .sqsh file
+   -  List the currently available images in the shared .sqsh file cache
+
+Download the shared cache .sqsh file for the current default Determined CUDA and CPU images (enroot
+import), and then create the associated containers from them for the current user (enroot create)
+use the following command:
+
+   .. code:: bash
+
+      manage-enroot-cache -s /shared/enroot --cuda --cpu
+
+Download the shared cache .sqsh file for an arbitrary docker image (enroot import), and then create
+a container from it for the current user (enroot create) use the following command:
+
+   .. code:: bash
+
+      manage-enroot-cache -s /shared/enroot determinedai/environments:cuda-10.2-base-gpu-mpi-0.19.4
+
+List the currently available images in the {sqshDir} cache with:
```

Comment:
`{sqshDir}`?

We don't actually define it, and it isn't referenced below.
should it instead by `/shared/enroot` directory?